### PR TITLE
add zh translation for installation

### PR DESCRIPTION
**Pull request by yyLeaves**
# What does this PR do?
- Add zh (Chinese) translation for installation section #20095
- Add installation to _toctree.yml

## Who can review?
Documentation: @sgugger @stevhliu and @MKhalusova
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by stevhliu — APPROVED** [review]
Thanks for your translation! 🤗 

For some reason, the second-level headings aren't being recognized so we're going to look into that. Once it gets resolved, we'll be free to merge!

**Comment by yyLeaves** [comment]
> Thanks for your translation! 🤗
> 
> For some reason, the second-level headings aren't being recognized so we're going to look into that. Once it gets resolved, we'll be free to merge!

Noted with thanks! @stevhliu

File: docs/source/zh/installation.md
Comment (wfjsw) [inline docs/source/zh/installation.md:150]: Extra spacing may be desired here:

image

File: docs/source/zh/installation.md
Comment (wfjsw) [inline docs/source/zh/installation.md:58]: Inconsistent spacing between CJK and ASCII characters. 

Generally in web typesetting environments, it may be more comfortable to read by inserting spaces between those CJK and ASCII characters. Microsoft Word do it automatically, but web browsers don't.

There are several other instances of this issues in other part of the docs.

Comment (stevhliu) [reply docs/source/zh/installation.md:58]: Thanks for your review @wfjsw, would you like to open a PR to fix this issue?

Comment (wfjsw) [reply docs/source/zh/installation.md:58]: @stevhliu Sure! I'll do it later after finish reviewing other files. Meanwhile could you take a look at #26603?

Comment (yyLeaves) [reply docs/source/zh/installation.md:58]: > Inconsistent spacing between CJK and ASCII characters.
> 
> Generally in web typesetting environments, it may be more comfortable to read by inserting spaces between those CJK and ASCII characters. Microsoft Word do it automatically, but web browsers don't.
> 
> There are several other instances of this issues in other part of the docs.

The justification for the space inconsistency is that  `🤗 Transformers` seems like a proper noun so I followed the format in [快速上手](https://huggingface.co/docs/transformers/v4.34.0/zh/quicktour). On a second review, neither `🤗 Transformers简介` nor `快速上手` section has a consistent translation format. Apart from the inconsistent space problem, they use a mix of Chinese and English punctuations (mostly English punctuations but Chinese punctuations would be more appropriate in my view). If possible, please also take into account these issues while fixing.

**Pull request merged**
